File: c++/control_flow/if_statement/if_else_statement.cpp
#include <iostream>

int main()
{
  if (1 > 2)
    std::cout << "1 > 2" << std::endl;
  else
    std::cout << "!(1 > 2)" << std::endl;

  if (2 > 1)
    std::cout << "2 > 1" << std::endl;
  else
    std::cout << "!(2 > 1)" << std::endl;

  std::cout << std::endl;

  bool condition;

  condition = false;

  if (condition)
    std::cout << "false: ok" << std::endl;
  else
    std::cout << "false: nope" << std::endl;

  condition = true;

  if (condition)
    std::cout << "true: ok" << std::endl;
  else
    std::cout << "true: nope" << std::endl;

  std::cout << std::endl;
}

File: java/control_flow/if_statement/IfElseStatement.java
class IfElseStatement {
  public static void main(String[] args) {
    if (1 > 2)
      System.out.println("1 > 2");
    else
      System.out.println("!(1 > 2)");

    if (2 > 1)
      System.out.println("2 > 1");
    else
      System.out.println("!(2 > 1)");

    System.out.println();

    boolean condition;

    condition = false;

    if (condition)
      System.out.println("false: ok");
    else
      System.out.println("false: nope");

    condition = true;

    if (condition)
      System.out.println("true: ok");
    else
      System.out.println("true: nope");

    System.out.println();
  }
}
